contributing/DOCS-CONTRIBUTING.md

```diff
@@ -68,45 +71,203 @@ documentation efforts, who are happy to help you get started.
-documentation most relevant to contributors of a component should be placed in
-the `docs` folder in a given component's repository. An example of this is the
-[spec](https://github.com/knative/serving/tree/master/docs/spec) folder in the
-Serving component.
-
-Code samples follow a similar strategy, where most code samples should be
-located in the `docs` repository. If there are code samples used for testing
-that are only expected to be used by contributors, those samples are put in the
-`samples` folder within the component repo.
+There are currently two general types of Knative docs, either contributor
+related content, or external-facing user content.
+
+#### Choosing the correct repo
+
+Depending on the type of content that you want to contribute, it might belong in
+one of the Knative code repositories (`knative/serving`, `knative/eventing`,
+etc.) or in `knative/docs`, the Knative documentation repo.
+
+* **Contributor focused content**
+
+  * *Documentation*: Includes content that is component specific and relevant
+    only to contributors of a given component. Contributor focused
+    documentation is located in the corresponding `docs` folder of that
+    component's repository. For example, if you contribute code to
+    the Knative Serving component, you might need to add contributor focused
+    information into the `docs` folder of the
+    [knative/serving repo](https://github.com/knative/serving/tree/master/docs/).
+
+  * *Code samples*: Includes contributor related code or samples. Code or samples
+    that are contributor focused also belong in their corresponding
+    component's repo. For example, Eventing specific test code is located in the
+    [knative/eventing tests](https://github.com/knative/eventing/tree/master/test)
+    folder.
+
+* **User focused content**
+
+  * *Documentation*: Includes all content for users of Knative. The
+    external-facing user documentation belongs in the
+    [`knative/docs` repo](https://github.com/knative/docs). All content in
+    `knative/docs` is published to [www.knative.dev](https://www.knative.dev).
+
+  * *Code samples*: Includes user facing code samples and their accompanying
+    step-by-step instructions. User code samples are currently separated into
+    two different locations within the `knative/docs` repo. See the following
+    section for details about determining where you can add your code sample.
+
+##### Determining where to add user focused code samples
+
+There are currently two categories of user focused code samples, either (1)
+*Knative owned and maintained*, or (2) *Community owned and maintained*.
+
+  * **Knative owned and maintained**: Includes code samples that are actively
+    maintained and e2e tested. With currency, clarity, and resource in mind,
+    only the code samples that meet the following requirements are located in
+    the `docs/[*component*]/samples` folders of the `knative/docs` repo:
```

Comment:
remove `*` for consistency or add to line 192

Reply:
done